@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1665,22 +1678,27 @@ export class TextElementsRenderer {
                 iconIndex === undefined
             );
```

Comment:
See my comments in Placement.ts. Probably we could skip placing the text if the icon is rejected. Makes the code simpler with 'placePointLabel'. We were placing the text just to mark it invisible instead of rejected to avoid the fade out, but I'm not sure it's worth the confusion it add to placement code.